lib/mllp/receiver.ex

```diff
@@ -62,10 +84,71 @@ defmodule MLLP.Receiver do
   @spec stop(any) :: :ok | {:error, :not_found}
   def stop(port) do
     receiver_id = get_receiver_id_by_port(port)
-
     :ok = :ranch.stop_listener(receiver_id)
   end
 
+  @doc """
+  A function which can be used to embed an MLLP.Receiver under Elixir v1.5+ supervisors.
+  Unlike `start/1`, `start/2`, or `start/3` this function takes two additional options : `ref` and `transport_opts`.
+  Note that if a `ref` option is not supplied a reference will be created for you using `make_ref/0`.
+      children = [{MLLP.Receiver, [
+          ref: MyRef,
+          port: 4090,
+          dispatcher: MyDispatcher,
```

Comment:
Can we change this to `MLLP.DefautDispatcher`

Reply:
Yup, it should be... you have hawk eyes my friend!